### DISCUSSION TITLE
Helm - Using runtimeConfigRef During initial "kind:Provider" Install

### DISCUSSION THREAD

**Discussion by tath81** [crossplane/crossplane#6129] [Q&A]
When deploying a Crossplane Provider resource via a Helm chart, the `runtimeConfigRef` field cannot be set during the initial installation. This is because the referenced ProviderConfig resource does not exist yet and must be created after the Provider is installed (CRDs). This creates a sequencing issue when attempting to manage the installation and configuration entirely through Helm, as Helm templates are typically applied all at once. If anyone experienced the same problem please comment. Thank you!

**Comment by jbw976** [comment]
This open issue may be relevant for your scenario/challenge too 😉 
* https://github.com/crossplane/crossplane/issues/5259

**Reply by tath81** [reply]
Yep, this fits the scenario/challenges I'm facing. When can we expect this to be supported, and are there examples I can use to unblock myself?
